samples/server/petstore/cpp-restbed/git_push.sh

```diff
@@ -1,11 +1,17 @@
 #!/bin/sh
 # ref: https://help.github.com/articles/adding-an-existing-project-to-github-using-the-command-line/
 #
-# Usage example: /bin/sh ./git_push.sh wing328 openapi-petstore-cpprest "minor update"
+# Usage example: /bin/sh ./git_push.sh wing328 openapi-pestore-perl "minor update" "gitlab.com"
```

Comment:
Are changes to this file intended? You didn't mention this in commit (though the changes itself look almost fine to me).

I think the usage example should say:
```openapi-petstore-restbed```

instead of
```openapi-pestore-perl```

Reply:
good spot, this file is generated from this template file
https://github.com/OpenAPITools/openapi-generator/blob/47e2c0d027d867de67633bbc9c0a5d7e1054a778/modules/openapi-generator/src/main/resources/cpp-restbed-server/git_push.sh.mustache#L4

it was committed 2 months ago by [this](https://github.com/OpenAPITools/openapi-generator/commit/8f13b88ed9c0734603762fcde794e5b17c5559f3), which blanket apply to every other generators too.. which is not good. But shall be fixed by another PR I guess

Reply:
You're right, so your PR looks good to me now!